### PR TITLE
Improve mobile development flow

### LINKED ISSUE #3107: Improve mobile app development flow
**Is your feature request related to a problem? Please describe.**

Would be nice to make the mobile app development flow easier by creating new yarn commands that set the needed environment variables for mobile app.

**Describe the solution you'd like**

We should add two new yarn commands to `package.json`:

- `yarn serve:mobile`
- `yarn build:mobile`

Both commands should set the followings environment variables using `cross-env`:

```
VUE_APP_GRAPHQL_URL=https://backend.lunie.io/v1/graphql
STARGATE=https://lcd.nylira.net
RPC=https://rpc.nylira.net
MOBILE_APP=true
```
We should update the README accordingly.

### PR DESCRIPTION
**Pull request by mariopino**
Closes #3107

**Description:**

Add two new yarn commands to package.json:

- `yarn serve:mobile`
- `yarn build:mobile`

Both commands set the environment variables needed by mobile app using cross-env:

```
VUE_APP_GRAPHQL_URL=https://backend.lunie.io/v1/graphql
STARGATE=https://lcd.nylira.net
RPC=https://rpc.nylira.net
MOBILE_APP=true
```

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,9 @@
   "author": "Lunie International Software Systems Inc. <hello@lunie.io>",
   "scripts": {
     "serve": "kill-port 9080 && vue-cli-service serve --port 9080",
+    "serve:mobile": "kill-port 9080 && cross-env VUE_APP_GRAPHQL_URL=https://backend.lunie.io/v1/graphql STARGATE=https://lcd.nylira.net RPC=https://rpc.nylira.net MOBILE_APP=true vue-cli-service serve --port 9080",
     "build": "vue-cli-service build",
+    "build:mobile": "cross-env VUE_APP_GRAPHQL_URL=https://backend.lunie.io/v1/graphql STARGATE=https://lcd.nylira.net RPC=https://rpc.nylira.net MOBILE_APP=true vue-cli-service build",
```
Comment (faboweb) [inline package.json]:
```suggestion
    "build:mobile": "cross-env MOBILE_APP=true npm run build",
```

File: package.json [hunk 2]
```diff
@@ -5,7 +5,9 @@
   "author": "Lunie International Software Systems Inc. <hello@lunie.io>",
   "scripts": {
     "serve": "kill-port 9080 && vue-cli-service serve --port 9080",
+    "serve:mobile": "kill-port 9080 && cross-env VUE_APP_GRAPHQL_URL=https://backend.lunie.io/v1/graphql STARGATE=https://lcd.nylira.net RPC=https://rpc.nylira.net MOBILE_APP=true vue-cli-service serve --port 9080",
```
Comment (faboweb) [inline package.json]:
```suggestion
    "build:mobile": "cross-env MOBILE_APP=true npm run serve",
```

**Comment by faboweb** [comment]
> ci/circleci: pendingUpdated

**Comment by jbibla** [comment]
there are no changees in bitrise here - so will `MOBILE_APP=true` even be set on the actual builds? is bitrise using `build:mobile`?

**Comment by faboweb** [comment]
No, this should be set in Bitrise.

**Comment by jbibla** [comment]
> should be set in Bitrise

@mariopino please address this

**Comment by faboweb** [comment]
@mariopino done

**Pull request merged**